protocols/request-response/tests/ping.rs

```diff
@@ -110,17 +110,15 @@ fn ping_protocol() {
                     } else {
                         req_id = swarm2.send_request(&peer1_id, ping.clone());
                     }
-                    println!("hi, {}", req_id);
                 },
                 e => panic!("Peer2: Unexpected event: {:?}", e)
             }
         }
 
-        // check if is_pending_outbound returns false for a finished request when we have already issued a new one
         let req_id2 = swarm2.send_request(&peer1_id, ping.clone());
```

Comment:
hi @mxinden, could you give me some pointers on how to come up with a test for this scenario?